### PR TITLE
Fix spinner not showing up (#223)

### PR DESCRIPTION
**Pull request by hugomflavio**

### CODE REVIEW

File: R/run.R [hunk 1]
```diff
@@ -277,10 +277,11 @@ run_manage <- function(proc, timeout, spinner, stdout_line_callback,
 
   spin <- (function() {
     state <- 1L
-    phases <- c("-", "\\", "|", "-")
+    phases <- c("-", "\\", "|", "/")
     function() {
       cat("\r", phases[state], "\r", sep = "")
-      state <<- (state + 1) %% length(phases) + 1L
+      state <<- (state)%%length(phases) + 1L
```
Comment (gaborcsardi) [inline R/run.R]: Can you please remove the parens around `state` and add spaces around `%%`? Thanks!

Comment (hugomflavio) [reply R/run.R]: I have edited the line as suggested

**Comment by hugomflavio** [comment]
> Looks good, could you please fix the code style (see comment) and add an entry in the `NEWS.md` file? THanks!

Added a new line in the NEWS.md describing the change

**Comment by gaborcsardi** [comment]
Great, thanks much!

**Pull request merged**